jacoco-maven-plugin/src/org/jacoco/maven/ReportAggregateMojo.java

```diff
@@ -168,10 +170,9 @@ public String getName(final Locale locale) {
 	private List<MavenProject> findDependencies(final String... scopes) {
 		final List<MavenProject> result = new ArrayList<MavenProject>();
 		final List<String> scopeList = Arrays.asList(scopes);
-		for (final Object dependencyObject : getProject().getDependencies()) {
-			final Dependency dependency = (Dependency) dependencyObject;
-			if (scopeList.contains(dependency.getScope())) {
-				final MavenProject project = findProjectFromReactor(dependency);
+		for (final Artifact artifact : getProject().getArtifacts()) {
```

Comment:
`MavenProject::getArtifacts()` returns a list of all resolved dependencies (transitive included).  `MavenProject::getDependences()` just returns the direct dependencies specified in the pom.